### DISCUSSION TITLE
Confusion in the "Visualizing categorical data / Estimating central tendency" doc page

### DISCUSSION THREAD

**Discussion by gcaria** [mwaskom/seaborn#3100] [Q&A]
In the [Visualizing categorical data](https://seaborn.pydata.org/tutorial/categorical.html) doc page, there is a bit of confusion in the `Estimating central tendency` section.  Throughout the section `catplot` is used in the snippets, but other functions are referred to in the text (`barplot`, `countplot`, `pointplot`, etc)

**Comment by mwaskom** [comment]
I think the key here is in the second paragraph

<img width="691" alt="image" src="https://user-images.githubusercontent.com/315810/196683290-8044e9ca-4d22-4d59-88e1-31fb653eb5cd.png">
